Library/src/Systems/Organizations/OrganizationSystem.cpp

```diff
@@ -214,21 +334,14 @@ void OrganizationSystem::BulkInviteToOrganization(const csp::common::Optional<cs
 	}
 	else
 	{
-		const csp::common::Array<csp::common::String>& CurrentOrganizationIds = GetCurrentOrganizationIds();
+		SelectedOrganizationId = GetCurrentOrganizationId();
 
-		if (CurrentOrganizationIds.Size() == 0)
+		if (SelectedOrganizationId.IsEmpty())
 		{
-			CSP_LOG_ERROR_MSG("OrganizationSystem::InviteToOrganization failed: You do not belong to an Organization.");
-
 			INVOKE_IF_NOT_NULL(Callback, MakeInvalid<NullResult>());
```

Comment:
How come we removed the error log message? Seems useful!

Reply:
Not sure. Agreed they are useful so I have added error logs for all of these.